### PR TITLE
collect k8s audit logs using host tailer CR

### PR DESCRIPTION
**Pull request by sittg**
- new host tailer for collecting audit logs from apiserver (collects logs from master nodes filesystem and pushes them to container stdout (k8s-audit-log)
- new priority class for hosttailer
- readme updated
- helm chart version logging-policies-1.02.tgz created

### CODE REVIEW

**Comment by bartos-martin** [comment]
with current setup, both HostTailer and PriorityClass will be created on ALL clusters (downstream AND rancher).  Is this OK? will it work even on clusters with audit turned off ?

**Comment by pstefka** [comment]
> with current setup, both HostTailer and PriorityClass will be created on ALL clusters (downstream AND rancher). Is this OK? will it work even on clusters with audit turned off ?

We talked about this with Tomas yesterday = possibility to turn off the creation of hosttailers (and priority class).. nevertheless he assured me, that running the workloads on a cluster without audit logs enabled shouldnt be a problem. Btw. in the target architecture, all clusters should have audit logging enabled.

Definitelly cleaner solution would be to have the possibility to turn off this feature though.

File: charts/logging-policies/templates/host-tailer-k8s-audit-log.yaml
Comment (pstefka) [inline charts/logging-policies/templates/host-tailer-k8s-audit-log.yaml:1]: > with current setup, both HostTailer and PriorityClass will be created on ALL clusters (downstream AND rancher). Is this OK? will it work even on clusters with audit turned off ?

@bartos-martin We talked about this with @sittg yesterday = possibility to turn off the creation of hosttailers (and priority class).. nevertheless he assured me, that running the workloads on a cluster without audit logs enabled shouldnt be a problem. Btw. in the target architecture, all clusters should have audit logging enabled.

Definitelly cleaner solution would be to have the possibility to turn off this feature though.

**Review by pstefka — APPROVED** [review]
Other than the option to disable the feature via variable, LGTM

**Review by bartos-martin — COMMENTED** [review]
all good

**Review by sedlak7 — APPROVED** [review]

**Pull request merged**